Tests/Common/Securities/SecurityPortfolioManagerTests.cs

```diff
@@ -2402,7 +2402,7 @@ public void InternalPartialCashSettledPutAssignmentReducesSomeCash()
             securities[Symbols.SPY].SetMarketPrice(new TradeBar { Time = securities.UtcTime, Symbol = Symbols.SPY, Close = 189 });
             securities[Symbols.SPY_P_192_Feb19_2016].Holdings.SetHoldings(1, -2);
 
-            transactions.AddOrder(new SubmitOrderRequest(OrderType.OptionExercise, SecurityType.Option, Symbols.SPY_P_192_Feb19_2016, -1, 0, 0, securities.UtcTime, ""));
+            transactions.AddOrder(new SubmitOrderRequest(OrderType.OptionExercise, SecurityType.Option, Symbols.SPY_P_192_Feb19_2016, 1, 0, 0, securities.UtcTime, ""));
```

Comment:
Maybe it would be more explicitly if the sign was the inverse of the holdings sign? Or directly the `- security.Holdings.Quantity` like the algo manager (and `/ 2` for those kind of partial tests)

Reply:
yes, I like this. When making these changes I knew they were _correct_ but they seemed arbitrary. Will update 👍